<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceToolbarOptionsPage</class>
 <widget class="QWidget" name="InterfaceToolbarOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>317</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="customize_toolbar_box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Customize Action Toolbar</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="toolbar_layout_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="edit_button_box" native="true">
        <layout class="QHBoxLayout" name="edit_box_layout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="add_button">
           <property name="toolTip">
            <string>Add a new button to Toolbar</string>
           </property>
           <property name="text">
            <string>Add Action</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="insert_separator_button">
           <property name="toolTip">
            <string>Insert a separator</string>
           </property>
           <property name="text">
            <string>Add Separator</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="button_box_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="up_button">
           <property name="toolTip">
            <string>Move selected item up</string>
           </property>
           <property name="icon">
            <iconset theme=":/images/16x16/go-up.png">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="down_button">
           <property name="toolTip">
            <string>Move selected item down</string>
           </property>
           <property name="icon">
            <iconset theme=":/images/16x16/go-down.png">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="remove_button">
           <property name="toolTip">
            <string>Remove button from toolbar</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolbar_layout_list</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>insert_separator_button</tabstop>
  <tabstop>up_button</tabstop>
  <tabstop>down_button</tabstop>
  <tabstop>remove_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
